pip/commands/cache.py

```diff
@@ -0,0 +1,36 @@
+from __future__ import absolute_import
+
+import logging
+import os
+
+from pip.basecommand import Command
+from pip.status_codes import SUCCESS, ERROR
+
+
+logger = logging.getLogger(__name__)
+
+
+class CacheCommand(Command):
+    """\
+    Show information about pip's wheel cache.
+
+    Subcommands:
+        location: Print the location of the wheel cache."""
+    actions = ["location"]
+    name = "cache"
+    usage = """
+      %%prog [options] %s""" % "|".join(actions)
+    summary = "Show information about pip's wheel cache."
+
+    def run(self, options, args):
+        if not args or args[0] not in self.actions:
+            logger.warning(
+                "ERROR: Please provide one of these subcommands: %s" %
+                ", ".join(self.actions))
+            return ERROR
+        method = self.__getattribute__("action_%s" % args[0])
+        return method(options, args[1:])
+
+    def action_location(self, options, args):
+        logger.info(os.path.join(options.cache_dir, "wheels"))
```

Comment:
We probably want this to be a bit more structured yea? I imagine something that outputs something more like:

```
Cache Locations:
    HTTP    = /Users/dstufft/Library/Caches/pip/http
    Wheels = /Users/dstufft/Library/Caches/pip/wheels
```

Or something like that? Maybe It'd be better to switch ``pip cache locations`` for ``pip cache info`` and have somehting like:


```
HTTP Cache:
    Location = /Users/dstufft/Library/Caches/pip/http
    Size     = 284M
    Items    = 1045

Wheel Cache
    Location = /Users/dstufft/Library/Caches/pip/wheels
    Size     = 95M
    Items    = 391
```

I guess the key point here is that we currently have at least two caches, might add more in the future and arbitrarily picking one of them to return as the location feels like the wrong thing to do so we should at least support something that returns them all. Adding more information like Size and number of items might be interesting too to give people an idea of the scope of their caches.